anaconda.py

```diff
@@ -628,14 +628,24 @@ def _earlyExceptionHandler(ty, value, traceback):
     #   key are available
     from pyanaconda.modules.common.util import is_module_available
     from pyanaconda.modules.common.constants.services import SUBSCRIPTION
+    from pyanaconda.modules.common.errors.subscription import KickstartRegistrationError
+    from pyanaconda.errors import errorHandler, ERROR_RAISE
+
+    def registration_error_handler(message):
```

Comment:
Please, move this function, for example, to `pyanaconda.ui.lib.subscription`. We are trying to make `anaconda.py` smaller.

Reply:
Sure, moved. :)